File: 1. Conditional Problems/Conditional Problems in Python/p2.py
# write a python program to check whether a number is divisible by 5 and 11 or not

n = int(input("Enter n: "))

if n % 5 == 0 and n % 11 == 0:
    print(f"{n}is divisisble by 5 and 11")
else:
    print(f"{n}given number is not divisisble by 5 and 11")

File: 1. Conditional Problems/Condional Problems in C/p2.c
// write a C program to check whether a number is divisible by 5 and 11 or not

#include <stdio.h>

int main()
{
    int n;
    printf("Enter n: ");
    scanf("%d", &n);

    n % 5 == 0 && n % 11 == 0 ? printf("%d is divisisble by 5 and 11",n) : printf("%d is not divisisble by 5 and 11",n);

    return 0;
}
